src/js/angular/import/controllers/import-view.controller.js

```diff
@@ -425,28 +453,30 @@ importViewModule.controller('ImportViewCtrl', ['$scope', 'toastr', '$interval',
             subscriptions.forEach((subscription) => subscription());
         };
 
+        const onActiveTabChanged = (activeTabId) => {
+            $scope.activeTabId = activeTabId;
+            ImportContextService.updateResources([]);
+            ImportContextService.updateShowLoader(true);
+            $scope.updateListHttp(true).finally(() => ImportContextService.updateShowLoader(false));
+        }
+
         const initSubscriptions = () => {
             subscriptions.push($scope.$on('repositoryIsSet', $scope.onRepositoryChange));
             subscriptions.push($scope.$on('$destroy', () => $interval.cancel(listPollingHandler)));
-            subscriptions.push(ImportContextService.onActiveTabIdUpdated((newActiveTabId) => {
-                $scope.activeTabId = newActiveTabId;
-                ImportContextService.updateResources([]);
-                ImportContextService.updateShowLoader(true);
-                $scope.updateListHttp(true).finally(() => ImportContextService.updateShowLoader(false));
-            }));
+            subscriptions.push(ImportContextService.onActiveTabIdUpdated((newActiveTabId) => onActiveTabChanged(newActiveTabId)));
             $scope.$on('$destroy', removeAllListeners);
         };
 
         // =========================
         // Initialization
         // =========================
-
-        // TODO: Beware that this init is called tree times due to the child controllers which extends this one. We should refactor this.
-        const init = () => {
+        $scope.importViewControllerInit = () => {
+            loadDefaultSettings();
+            pollList();
```

Comment:
I wonder if these two calls above: the `pollList()` and the `loadDefaultSettings()` shouldn't be moved below subscriptions and loading the app data, or even chanined to getAppData maybe?

Reply:
I will try

Reply:
Done